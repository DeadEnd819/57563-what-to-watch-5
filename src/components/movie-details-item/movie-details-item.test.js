import React from "react";
import renderer from "react-test-renderer";
import MovieDetailsItem from "./movie-details-item";

it(`Should MovieDetailsItem render correctly`, () => {
  const tree = renderer
    .create(
        <MovieDetailsItem
          name={`Kate Muir`}
          value={`The mannered, madcap proceedings are often delightful`}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});


