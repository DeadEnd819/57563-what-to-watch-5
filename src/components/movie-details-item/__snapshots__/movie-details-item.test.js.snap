// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieDetailsItem render correctly 1`] = `
<p
  className="movie-card__details-item"
>
  <strong
    className="movie-card__details-name"
  >
    Kate Muir
  </strong>
  <span
    className="movie-card__details-value"
  >
    The mannered, madcap proceedings are often delightful
  </span>
</p>
`;
